### ISSUE TITLE
Approve request from @patcon to allow org access

### ISSUE THREAD

**Issue by patcon** [PDIS/holopolis.pdis.tw#5]
Ok, so it seems that in order to set up heroku review apps (as part of https://github.com/PDIS/holopolis.pdis.tw/pull/4), heroku needs access to _my_ access to the github org:

![screenshot of heroku app's drpdown to link github repo](https://imgur.com/Py23Kda.png)

Any chance you'd be able to add me to the org? I might need to be on a new team that has access to the holopolis.pdis.tw repo... sorry! this seems like a bunch of hoops... i'm used to being github org owner, so this pain point is new to me :)

cc: @shuyanglin

**Comment by patcon** [comment]
Alternatively, we could do a quick call, and I could walk through giving you owner access to the heroku app, and we could configure the review apps at the same time :)

**Comment by audreyt** [comment]
I just granted the oauth app access to you — thanks for the contribution!

+1 on a quick-call and giving owner access to shuyang at some point

**Comment by patcon** [comment]
Perf. Thanks. Added a call to other repos TODOs

PDIS org is still not showing as an option to connect in the heroku interface. I've sleuthed the other orgs that DO appear in the dropdown, and logic seems to be that:
- I need to a be a member of the org when it's an org repo
- [I suspect that] I need to be on an org team that is assigned access to repo

Is this possible? Sorry, this is a bit of a pain, but i feel like it's worth navigating and then documenting in the interest of making this less painful for the next external participant :)
